packages/logging-controller/src/LoggingController.ts

```diff
@@ -0,0 +1,101 @@
+import {
+  BaseController,
+  BaseConfig,
+  BaseState,
+} from '@metamask/base-controller';
+import { v1 as random } from 'uuid';
+
+export enum LogType {
+  eth_sign_proposed = 'eth_sign_proposed',
```

Comment:
We don't have this ESLint rule enabled yet, but as a rule we use PascalCase for enum property names. So to fit idiomatically with other enums, we ought to use:

``` typescript
export enum LogType {
  EthSignProposed = 'eth_sign_proposed',
  EthSignSigned = 'eth_sign_signed',
  // ...
}
```

Reply:
I updated these types and data structure quite a bit